### ISSUE TITLE
Uncaught TypeError: e.type(...).map is not a function

### ISSUE THREAD

**Issue by ProLoser** [mulesoft/api-workbench#131]
1. Open Console
2. Click on 'View 2 more resources'
3. Clicked on 'View n more resources' again (for nested resources)

**Atom Version**: 1.5.3
**System**: Mac OS X 10.10.5
**Thrown From**: [api-workbench](https://github.com/mulesoft/api-workbench) package, v0.8.20

### Stack Trace

Uncaught TypeError: e.type(...).map is not a function

```
At /Users/dsofer/.atom/packages/api-workbench/main.js:25

TypeError: e.type(...).map is not a function
    at t.render (/Users/dsofer/.atom/packages/api-workbench/main.js:25:10678)
    at [object Object].ReactCompositeComponentMixin._renderValidatedComponentWithoutOwnerOrContext (/Users/dsofer/.atom/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:789:34)
    at [object Object].ReactCompositeComponentMixin._renderValidatedComponent (/Users/dsofer/.atom/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:816:14)
    at [object Object].ReactCompositeComponentMixin.mountComponent (/Users/dsofer/.atom/packages/api-workbench/node_modules/react/lib/ReactCompositeComponent.js:237:30)
    at Object.ReactReconciler.mountComponent (/Users/dsofer/.atom/packages/api-workbench/node_modules/react/lib/ReactReconciler.js:38:35)

**Comment by ProLoser** [comment]
After some digging I realized the raml file is not being parsed properly, the resource is `    /recently_aired/filters/content_restriction/{content_restriction_key}/from/{date_from}/to/{date_to}/sort/{sort_method}/offset/{offset}/size/{size}:`

<img width="738" alt="screen shot 2016-02-26 at 4 00 37 pm" src="https://cloud.githubusercontent.com/assets/67395/13368752/1f837b8a-dca2-11e5-9b0a-67603a84687c.png">

Note the lower resource is not red (syntax highlighting failure) and the indentation is 1 space off and rendering strangely.

**Comment by ProLoser** [comment]
After exploring, it seems the parser does not like more than 3 uriTokens in a resource endpoint path (not including ancestor uriTokens).

**Comment by ProLoser** [comment]
However, although the parser is breaking and throwing lots of errors, removing this endpoint did not fix the api console bug reported in this issue.

**Comment by ddenisenko** [comment]
A quick test for more than 3 uri parameter in a single resource: 
1) Parser eats that.
2) API Workbench coloring does not.

Could you please attach a file or post a code snapshot that makes console to break? Couldnt reproduce the console exception with just what is on screenshot.

**Comment by nowthen** [comment]
Can also be reproduced by having a trait.
1. Open Console
2. Click on GET for the request with a trait

![image](https://cloud.githubusercontent.com/assets/17931624/13887477/411d18f2-ed12-11e5-83b9-97959492081d.png)

**Comment by simlegate** [comment]
![8777777](https://cloud.githubusercontent.com/assets/2359726/14410514/26e89b3c-ff64-11e5-87a2-c221e6c37a90.png)

I have encountered the issue

Atom Version: 1.6.2
System: Mac OS X 10.9
Thrown From: api-workbench package, v0.8.23

**Comment by sichvoge** [comment]
Thanks everyone for reporting and confirming the issue. As @ddenisenko asked, could anyone please provide a RAML file that we can use on our side to reproduce the issue.

**Comment by ddenisenko** [comment]
Lots of reports here. Most probably, all of those are fixed in 0.8.25, though I could not reproduce the first one. Let me know how it goes.

**Comment by simlegate** [comment]
I have tried the new version and the issue has been fixed.
@ddenisenko  Thank for your hard work.
